main.go

```diff
@@ -88,7 +88,17 @@ func realmain(arguments []string) int {
 	}
 	env.Decode(&flags)
 
-	confFile, err := configload.LoadFile(flags.FilePath)
+	if cmd == "verify" {
+		log := newLogger(flags.LogFormat, flags.LogLevel, flags.LogPretty)
+
+		err := command.NewCommand(ctx, cmd).Execute(command.Args{flags.FilePath}, nil, log)
```

Comment:
```suggestion
		err = command.NewCommand(ctx, cmd).Execute(command.Args{flags.FilePath}, nil, log)
```

Reply:
log returned err here with `WithError()` instead of returning empty ones.

Reply:
No, the current IF block returns 1 or 0. The logger is never reached.